index.html

```diff
@@ -0,0 +1,201 @@
+                    <option value="Dublin">
+                    <option value="London">
+                    <option value="Amsterdam">
+                    <option value="Berlin">
+                    <option value="Kharkiv">
+                </datalist>
+            </form>
+            <form class="dates" action="" method="get">
+                <label for="departure_date">Leave <img src="Images/logos/leave plane.png" alt="logo"></label>
+                <input type="date" id="departure_date" name="departure_date">
+                <img src="Images/logos/Planner.png" alt="logo">
+                <label for="return_date">Leave <img src="Images/logos/return plane.png" alt="logo"></label>
+                <input type="date" id="return_date" name="return_date">
+            </form>
+            <input class="search_button" type="submit" value="">
+        </div>
+    </div>
+    <main>
+        <h3 class="offers_header">Special Upcoming Offers</h3>
+        <div class="group_type">
+            <span>Team</span>
+            <span>Couple</span>
+            <span>Family</span>
+        </div>
+        <div class="special_offers">
+            <div class="offer_block">
+                <img src="Images/image/Loga sea.png" alt="Loga sea">
+                <p class="date">14 FEB 2022</p>
+                <p><span class="relax">Relax</span> 3 Days, 3 Nights</p>
+                <h4>Loga Sea</h4>
+                <p><b>700$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+            <div class="offer_block">
+                <img src="Images/image/Ansgar Scheffold.png" alt="Ansgar Scheffold">
+                <p class="date">18 JUN 2022</p>
+                <p><span class="adventure">Adventure</span> 4 Days, 3 Nights</p>
+                <h4>Ansgar Scheffold</h4>
+                <p><b>400$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+            <div class="offer_block">
+                <img src="Images/image/Lona X.png" alt="Lona X">
+                <p class="date">22 DEC 2022</p>
+                <p><span class="relax">Relax</span> 7 Days,6 Nights</p>
+                <h4>Lona X</h4>
+                <p><b>340$</b>/Person</p>
+                <button>Book Now</button>
+            </div>
+        </div>
+        <div class="about_us">
+            <h3 class="about_us_header"> Travel Any Corner of The World With Us</h3>
+            <p class="about_us_text"> Would you explore nature paradise in the world, let’s find the best destination in world with us, Would you explore nature paradise in the world, let’s find the best destination in world with us. Would you explore nature paradise in the world, let’s find the best destination in world with us.
+
+                Would you explore nature paradise in the world, let’s find the best destination in world with us.</p>
+            <img class="about_us_img" src="Images/image/Travel blog.jpg" alt="Image">
```

Comment:
alt and title attributes should describe exact image and tell user what it should be. Usually it's fallback for cases when image didn't downloaded yet due to internet connection. Do not use long description. Not more then 5-20 symbols

Reply:
done